src/main/java/com/exasol/releasedroid/usecases/validate/StructureValidator.java

```diff
@@ -5,9 +5,9 @@
 /**
  * This interface provides repository-related validations.
  */
-public interface RepositoryValidator {
+public interface StructureValidator {
```

Comment:
I find it a bit hard to understand what's the purpose of the interface is just by looking at the name.
Maybe we can roll this back